helpers/helm-tester/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM python:3.7
 
-ENV HELM_VERSION=2.17.0
+ENV HELM_VERSION=3.4.1
 
-RUN wget --no-verbose https://storage.googleapis.com/kubernetes-helm/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
+RUN wget --no-verbose https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
     tar xfv helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
     mv linux-amd64/helm /usr/local/bin/ && \
-    rm -rf linux-amd64 && \
-    HOME=/ helm init --client-only && \
-    chmod 777 -R /.helm
+    mkdir --parents --mode=777 /.config/helm && \
+    HOME=/ helm repo add stable https://charts.helm.sh/stable && \
+    rm -rf linux-amd64
```

Comment:
nit should we also remove `*.tar.gz`

Reply:
nice catch, I don't know why this wasn't already cleaned => 902dd6c